packages/enzyme-adapter-react-15/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "enzyme-adapter-react-15",
+  "version": "2.9.1",
```

Comment:
these versions should all start at v0.0.0 until published, when they should jump to v1.0.0.